AllASPFuncs.py

```diff
@@ -827,50 +845,47 @@ def makeGrid(adaptive_grid, images, size, ra, dec, cutout_wcs_list,
 
 def plot_lc(fileroot):
 
-    fluxdata = pd.read_csv('./results/lightcurves/'+str(fileroot)+'_lc.csv')
-    supernova = fluxdata['true_flux']
-    measured_flux = fluxdata['measured_flux']
+    fluxdata = pd.read_csv('./results/lightcurves/'+str(fileroot)+'_lc.ecsv',
```

Comment:
For a future issue, make this `./results/lightcurves/` more flexible and something that can be passed in more easily.  For now I might even suggest making the argument to `plot_lc` the full path name.